### DISCUSSION TITLE
10 September 2024 2200 UTC

### DISCUSSION THREAD

**Discussion by ktoddbrown** [ktoddbrown/SoilDRaH#45] [RegularCalls]
### Agenda

These calls are lead by @ktoddbrown and are generally open calls.
Please contact them for call in information.
The purpose of these meetings are to move forward current issues related to the development of Soil-DRaH.
This may include teaching, mutual accountability, code review, paired programming, or other activities.

## (5 | 2200) Check ins
- Who is here
- Note taking?
- How are you showing up? <https://feelingswheel.com/>

## (45 | 2205) Issues checkins and stuck to unstuck
- *Pull up open issues where do each issue stand?*
- *How can we help each other move from stuck to unstuck on a topic?*

## (5 | 2250) Backlog
- *What new issues need to be created?*
- *Are the current issues clear?*

**Comment by vaasukimarupaka** [comment]
## (5 | 2200) Check ins
Who is here: Vaasuki, Graham, Gabriel, Maggie, Brandon
Note taking? Vaasuki
How are you showing up? https://feelingswheel.com/
## (45 | 2205) Issues checkins and stuck to unstuck
- **Pull up open issues where does each issue stand?**

CPEAT #1: 
Vaasuki **requested review** of CPEAT annotations with checkAnnotations function and
the long table data frame. funky row stacking at end of table

NRCS-NCSS #5:
Brandon says Section 2 of report file could use more comments
Vaasuki- focus on NRCS markdown file and take a look at the function 

variable mapping #6:
Kathe suggests we explore https://github.com/mapping-commons/sssom (Brandon)
SSSOM and ROBOT will be useful, but not yet 

Longleaf pine biomass #16:
Gabriel **requests review** of `of_variable` 
Need verification of of_variable and annotations?

International Soil Radiocarbon Database v1.0 #17
Kathe suggests layer level SOC variable mapping? (over my head)
Kate is currently working on it, hopefully we will more updates going forward

- [ ]  Geochemical and mineralogical data #20
			- [ ] Gabriel **requests review** of `of_variable` mappings
- [ ] Rename annotations and mapping files #36
			- [ ] Kathe suggests renaming these files so they can be sorted since nesting is not possible in packages


## (5 | 2250) Backlog
What new issues need to be created?
Are the current issues clear?

## What folks are doing going forward:
Revisit ISCN4 data integration 

Gabriel- creating how-to's for our lab's annotation generation process and well as the readX functions. Getting more accustomed to R library's functions that we use in this lab in the process.
Maggie- working on familiarizing with git and coworkers with Gabriel. 
Graham- will join the co-working calls and work with Kathe on long leaf pine and Smith 2019 review.
Vaasuki- will work on NRCS documentation and read function.
Brandon- try and close the gap between SSSOM & ROBOT for mapping tables. Will try joining co-working hours to discuss further.

## (5 | 2255) Check outs [appreciations, discoveries, and ToDo]
Had to end the call early due to networks issues but folks know what they are doing next!
